### PR TITLE
update readme with instructions on how to enable events

### PR DESCRIPTION
**Pull request by pnzrr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,12 +26,16 @@ The maven build can be triggered by running `mvn clean install`. It uses the sha
...
 2. The optional `N` value in the key is relevant to the second important utility. The `EventListenerProviderFactory` implementations are all subclasses of `MultiEventListenerProviderFactory`, which enables multiple `EventListenerProvider` instances of the same type to run with different configurations. This is a facility that is not currently available in Keycloak, although some tickets and features in the future Admin UI indicate that it is coming soon.
 
+Most events require being enabled through the Admin UI. Go to (Configure) Realm Settings > Events (tab) > Event listeners. In the Event listeners dropdown select `ext-event-********` for the appropriate event being enabled.
```
Comment (pnzrr) [inline README.md:33]: Added

File: README.md [hunk 2]
```diff
@@ -123,36 +136,37 @@ public class MyUserAddRemove extends UserEventListenerProviderFactory {
...
+This provides the entities and REST endpoints required to allow webhook subscriptions to events. The events have been slightly modified so that there are no longer 2 types of events, but are now distinguished by a type prefix. Definition on the event format and types is available in the [Phase Two](https://phasetwo.io/) documentation under [Audit Logs](https://phasetwo.io/docs/audit-logs/).
+
+Webhooks are sent using the same mechanics as the `HttpSenderEventListenerProvider`, and there is an automatic exponential backoff if there is not a 2xx response. The sending tasks are scheduled in a thread pool and executed after the Keycloak transaction has been committed.
 
-Webhooks are sent using the same mechanics as the `HttpSenderEventListenerProvider`, and there is an automatic exponential backoff if there is not a 2xx response. The sending tasks are scheduled in a thread pool and executed after the Keycloak transaction has been committed. 
+Enable webhook events in the Admin UI by going to (Configure) Realm Settings > Events (tab) > Event Listeners, in the Event listeners dropdown select `ext-event-webhook` and save.
```
Comment (pnzrr) [inline README.md:143]: added

Comment (xgp) [reply README.md:143]: Why remove the first sentence?

Comment (pnzrr) [reply README.md:143]: It didn't, the spacing just got picked up as Git as a removal, but its still there above.

Comment (pnzrr) [reply README.md:143]: image

**Review by xgp — APPROVED** [review]

**Pull request merged**